enterprise/backend/src/metabase_enterprise/representations/common.clj

```diff
@@ -0,0 +1,9 @@
+(ns metabase-enterprise.representations.common
+  (:require
+   [metabase-enterprise.representations.v0.core :as v0]))
+
+(defn toucan-model
+  "Given a representation, returns the internal Toucan model keyword we will use when importing to our app db."
+  [& {:keys [version type]}]
```

Comment:
should this stuff use `mu/defn` and have Malli schemas around it? Probably a bug to call this with a map with a `nil` `:type`, right?